### ISSUE TITLE
Missing ValidateDCIMChoice function

### ISSUE THREAD

**Issue by SheffSix** [benclaussen/NetboxPS#40]
If I specify the -form_factor parameter on any given CMDLET (Get-NetboxDCIMInterface is one example), I get the following error:

```
ValidateDCIMChoice: The term 'ValidateDCIMChoice' is not recognized as a name of a cmdlet, function, script file, or executable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
```

I can't see this function anywhere in the code, and it looks like a lot of references to it are commented out. Has it been removed? How do we specify the type of interface?

**Comment by benclaussen** [comment]
I apologize, yes I had to remove this a while ago due to changes in the API (Netbox got rid of the "choices"). This call was actually validating `status` parameter in `GetNetboxDCIMInterface`. I moved to letting the server reject an invalid status instead of doing it before sending the request.

Specifically, lines [76-78 ](https://github.com/benclaussen/NetboxPS/blob/ddf0a081b4e38d057cbd6edaf6c3eec0930b888c/Functions/DCIM/Devices/Get-NetboxDCIMDevice.ps1#L76) for that function. Removing those lines should fix the error.

I will do my best to work on that ASAP (but also not opposed to a PR 😋)
